src/unistd.rs

```diff
@@ -2463,8 +2463,11 @@ mod setres {
     ///
     /// Err is returned if the user doesn't have permission to set this UID.
     #[inline]
-    pub fn setresuid(ruid: Uid, euid: Uid, suid: Uid) -> Result<()> {
-        let res = unsafe { libc::setresuid(ruid.into(), euid.into(), suid.into()) };
+    pub fn setresuid(ruid: Option<Uid>, euid: Option<Uid>, suid: Option<Uid>) -> Result<()> {
```

Comment:
If setresuid is going to take Option arguments now, then the docs should say "and/or saved uid".

Reply:
my bad, I did not see this pattern elsewhere. Would it better to document ?

```
/// Sets the real, effective, and saved gid.
/// If one of the arguments equals None, the corresponding value is not changed.
...
```

Reply:
Yeah, that would work.